files/generate-CA-and-clientcerts.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This looks generally fine, though I've generally found it easier to use a ``openssl.conf`` file instead of passing all of the options here as command line flags.  No difference in terms of the resulting certs though.